app/assets/stylesheets/ama_layout/layout_components/breadcrumbs.scss

```diff
@@ -5,4 +5,16 @@
     display: inline-block;
     font-size: 0.75rem;
   }
+
+  a.disabled{
+    pointer-events: none;
+    color: $stormcloud;
+  }
+
+  li:not(:last-child)::after {
```

Comment:
Actually, it doesn't work here when selecting the element based on class. Foundation uses the same selector to make the separator work. We have to override the same selector that Foundation uses.